From 5f3f69533a2298cc0f2b45e17184c479b0708687 Mon Sep 17 00:00:00 2001
From: Matus Honek <mhonek@redhat.com>
Date: Thu, 16 Jan 2020 12:21:45 +0100
Subject: [PATCH] Issue 50824 - dsctl remove fails with "name 'ensure_str' is
 not defined"

Bug Description:
Missing import since commit c39c7bb.

Fix Description:
Add the import.

Fixes https://pagure.io/389-ds-base/issue/50824

Author: Matus Honek <mhonek@redhat.com>

Review by: Mark (thanks!)

(cherry picked from commit 4f9aafca9a9927812da5e37ce71d79d1fd23b25a)
---
 src/lib389/lib389/instance/remove.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lib389/lib389/instance/remove.py b/src/lib389/lib389/instance/remove.py
index 7a69da514..77f5a45f7 100644
--- a/src/lib389/lib389/instance/remove.py
+++ b/src/lib389/lib389/instance/remove.py
@@ -11,7 +11,7 @@ import shutil
 import subprocess
 import logging
 from lib389.nss_ssl import NssSsl
-from lib389.utils import selinux_label_port, assert_c, ensure_list_str
+from lib389.utils import selinux_label_port, assert_c, ensure_str, ensure_list_str
 
 
 ######################## WARNING #############################
-- 
2.21.1

